main/test/ca/uwaterloo/flix/library/TestChain.flix

```diff
@@ -375,16 +375,16 @@ namespace TestChain {
 
     @test
     def repeat01(): Bool = Chain.repeat(-1, 1) == Chain.empty()
-    
+
```

Comment:
I think maybe github deleted all the indents with no line

Reply:
OK, that is fine.